src/new-client/src/Finsemble/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { StrictMode } from "react"
+import ReactDOM from "react-dom"
+import { GA_TRACKING_ID } from "@/constants"
+import GlobalStyle from "@/theme/globals"
+import { GlobalScrollbarStyle, ThemeProvider } from "@/theme"
+import { FinsembleApp } from "./FinsembleApp"
+
+import { connectToGateway } from "@adaptive/hydra-platform"
+import { noop } from "rxjs"
+
+if (!import.meta.env.VITE_MOCKS) {
+  connectToGateway({
+    url: import.meta.env.VITE_HYDRA_URL as string,
+    interceptor: noop,
+    useJson: true,
+  })
+}
+
+ReactDOM.render(
+  <StrictMode>
+    <GlobalStyle />
+    <ThemeProvider>
+      <GlobalScrollbarStyle />
+      <FinsembleApp />
+    </ThemeProvider>
+  </StrictMode>,
+  document.getElementById("root"),
+)
+
+const { ga } = window
+
+ga("create", {
+  trackingId: GA_TRACKING_ID,
+  transport: "beacon",
+})
+
+ga("set", {
+  dimension1: "finsemble",
```

Comment:
Do we have the different platform names in an `Enum` somewhere? Might be worth adding if not

Reply:
I don't know .. @garyadaptive ?